src/python/pants_release/git.py

```diff
@@ -36,3 +54,29 @@ def git_fetch(rev: str) -> str:
     """Fetch rev (e.g. branch or a SHA) from the upstream repository and return its SHA."""
     git("fetch", MAIN_REPO, rev)
     return git_rev_parse("FETCH_HEAD")
+
+
+def github_repo() -> github.Repository.Repository:
+    # Borrow the token from `gh`: `gh auth login` is far more convenient for interactive use, and
+    # setting `GH_TOKEN=...` env var works well for scripting use
```

Comment:
I don't get the bit about the env var, since we're not using it anymore.

Reply:
Oh you're assuming the CLI uses it?

Int hat case the right env var is `GITHUB_TOKEN`: https://cli.github.com/manual/

Reply:
Although https://cli.github.com/manual/gh_auth_login lists `GH_TOKEN` WTF?!

OK I think `GH_TOKEN` is likely the right answer.

Reply:
So to wrap up the initial comment. Mind typing together how the env var comes into play?

Reply:
The linked "environment variable" page explains that both are supported:

> GH_TOKEN, GITHUB_TOKEN (in order of precedence): an authentication token for github.com API requests. Setting this avoids being prompted to authenticate and takes precedence over previously stored credentials.

It's annoying that they're inconsistent about which one to use. I'll tweak the comment to be clearer.